Dockerfile

```diff
@@ -1,19 +1,43 @@
-FROM node:22-slim AS base
-ENV PNPM_HOME="/pnpm"
-ENV PATH="$PNPM_HOME:$PATH"
-RUN corepack enable
+# install and optimize dependencies
+FROM zenika/alpine-chrome:with-node AS base
+
 WORKDIR /app
+
+# copy package files
 COPY package.json pnpm-lock.yaml ./
-RUN corepack prepare
 
-FROM base AS prod-deps
-RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --prod --frozen-lockfile
+# install pnpm as root,
+# then switch back to chrome user
+USER root
+RUN npm install -g pnpm@10
```

Comment:
Lets use specific version, with all 3 numbers. prevents unwanted breaks in future